com.unity.visualeffectgraph/Editor/GraphView/Views/Controller/VFXViewControllerUndo.cs

```diff
@@ -203,6 +203,7 @@ private void SynchronizeUndoRedoState()
                     m_reentrant = true;
                     ExpressionGraphDirty = true;
                     model.GetOrCreateGraph().UpdateSubAssets();
+                    EditorUtility.SetDirty(graph);
```

Comment:
Interesting, I observed this behavior locally (without your change) :
![_undo_test_missing_dirty](https://user-images.githubusercontent.com/22494706/129858434-fedd5ef4-0933-4f8d-9704-62a3d2dd28b1.gif)
Was there another case where it causes an issue ?